presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1143,13 +1142,6 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
 
         List<PartitionUpdate> partitionUpdates = getPartitionUpdates(fragments);
 
-        if (isWritingStagingFilesEnabled(session)) {
-            stagingFileCommitter.commitFiles(session, handle.getSchemaName(), handle.getTableName(), partitionUpdates);
```

Comment:
I wonder if the SPI change that exposes this method to the engine should also be included in this PR. This will create better visibility of what is tried to be done here.